plugins/modules/provision_workflow_manager.py

```diff
@@ -911,6 +906,21 @@ class instance for further use.
             self.log("The provisioned status of the wired device is {0}".format(status), "INFO")
 
             if status == "success":
+                if to_force_provisioning is False:
+                    self.result["changed"] = False
+                    msg = "Device '{0}' is already provisioned.".format(self.validated_config.get("management_ip_address"))
+                    self.result['msg'] = msg
+                    self.result['diff'] = self.want
```

Comment:
Not needed we can remove and stick to the msg and response only.

Reply:
updated the code